lib/utils/scenario-manager.js

```diff
@@ -47,7 +47,12 @@ module.exports = CoreObject.extend({
     var pkgs = Object.keys(scenario.dependencies);
 
     pkgs.forEach(function(pkg){
-      bowerJSON.dependencies[pkg] = scenario.dependencies[pkg];
+      if (bowerJSON.devDependencies[pkg]) {
+        bowerJSON.devDependencies[pkg] = scenario.dependencies[pkg];
```

Comment:
I would keep them separated, so that a scenario can be expressed as follows:

```js

{
      name: 'ember-1.11',
      dependencies: {
        'ember': '~1.11.0'
      },
      devDependencies: {
        'somePkg': '~99.99.1'
      }
}